articles/remapping.md

```diff
@@ -0,0 +1,77 @@
+### Remapping Names
+#### ROS 1
+In ROS 1 [Names](http://wiki.ros.org/Names) can be remapped to other names by passing [remapping arguments](http://wiki.ros.org/Remapping%20Arguments) via the command line. `roscpp` additionally allows remaps to be passed via Init as strings in STL containers. A ROS1 remap consists of two names: one that should be replaced with another.
+
+In ROS1 remapping works with FQN. When a name is to be checked for remappings, it is first expanded to a FQN. Both parts of the remapping rule are expanded to a FQN. Then a name is changed to the remap's second name only if it exactly matches the first name.
```

Comment:
Yup, you're right. I never noticed that it processed them this way. Probably because the effect is the same in pretty much every case I can think of. Thanks for tracking down the citations, I'd still recommend putting the citations in the document, so if someone else comes along with the same question as me it is clear to them too.